tape/src/main/java/com/squareup/tape/QueueFile.java

```diff
@@ -54,6 +55,9 @@
 public class QueueFile {
   private static final Logger LOGGER = Logger.getLogger(QueueFile.class.getName());
 
+  /** Preview size of element when there is a problem **/
```

Comment:
```suggestion
  /** Preview size of element when there is a problem. */
```

[Comments should be sentences](https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html)